_includes/references.html

```diff
@@ -22,37 +22,37 @@ <h4>Publications</h4>
             <ol>
                 <li>
                     Martínez-Bazan, N., Muntés-Mulero, V., Gómez-Villamor, S., Nin, J., Sánchez-Martínez, M-A., and Larriba-Pey, J-L.
-                    DEX: High-performance Exploration on Large Graphs for Information Retrieval.
-                    In <i>Proc. 16th ACM Conference on Information and Knowledge Management</i>, pages 573–582, 2007.
+                    <a href="https://dl.acm.org/citation.cfm?id=1321521">DEX: High-performance Exploration on Large Graphs for Information Retrieval.</a>
+                    In <i>Proc. 16th ACM Conference on Information and Knowledge Management (CIKM)</i>, pages 573–582, 2007.
                 </li>
                 <li>
                     Rodriguez, M. A., and Neubauer, P.
-                    Constructions from Dots and Lines.
-                    <i>CoRR, abs/1006.2361</i>, 2010.
+                    <a href="http://onlinelibrary.wiley.com/doi/10.1002/bult.2010.1720360610/full">Constructions from Dots and Lines.</a>
+                    <i>Bulletin of the Association for Information Science and Technology</i>, August 2010.
                 </li>
                 <li>
                     Hartig, O.
-                    Reconciliation of RDF* and Property Graphs.
+                    <a href="https://arxiv.org/abs/1409.3288">Reconciliation of RDF* and Property Graphs.</a>
                     <i>CoRR, abs/1409.3288</i>, 2014.
                 </li>
                 <li>
-                    Junghanns, M., Petermann, A., Gómez, K., and Rahm, E.
-                    GRADOOP: Scalable Graph Data Management and Analytics with Hadoop.
-                    <i>CoRR abs/1506.00548</i>, 2015.
+                    Junghanns, M., Kießling, M., Averbuch, A., Petermann, A., and Rahm, E.
+                    <a href="https://dl.acm.org/citation.cfm?id=3078450">Cypher-based Graph Pattern Matching in GRADOOP.</a>
```

Comment:
Please revert this paper to the original one: "**GRADOOP: Scalable Graph Data Management and Analytics with Hadoop**", as this refers more to the PGM. There is a reference to "**Cypher-based Graph Pattern Matching in GRADOOP**" in the Cypher section (which is a more appropriate 'home' for it).